scripts/firedrake-install

```diff
@@ -578,6 +580,31 @@ def build_and_install_h5py():
     os.chdir("..")
 
 
+def build_and_install_slepc():
+
+    try:
+        petsc_dir = check_output(python + ["-c", "import petsc; print petsc.get_petsc_dir()"]).strip()
+        petsc_arch = ""
+    except subprocess.CalledProcessError:
+        raise InstallError("Unable to find installed PETSc when building SLEPc")
+
+    env = dict(os.environ)
+    env["PETSC_ARCH"] = petsc_arch
+    env["PETSC_DIR"] = petsc_dir
+
+    git_clone("git+https://bitbucket.org/slepc/slepc.git")
+    os.chdir("slepc")
```

Comment:
There is no point in this line (and 598, 602, 604). You don't need to chdir into slepc just to emit the log line.